respond.src.js

```diff
@@ -267,7 +267,11 @@ window.matchMedia = window.matchMedia || (function( doc, undefined ) {
 			}
 			
 			//inject active styles, grouped by media type
+			var keys = [];
 			for( var k in styleBlocks ){
+				keys.push( k );
+			}
+			for( var l = keys.length; l-- > 0;){
 				if( styleBlocks.hasOwnProperty( k ) ){
```

Comment:
`k` here is the value of the last iteration of the above loop. Must be `k = keys[l]` instead.